### PR TITLE
chores: Update build.yml

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

## Proposed changes

This pull request proposes the upgrade of SvelteKit to version 1.0. The latest version brings a myriad of improvements and bug fixes that will greatly improve our current development process. This version has been well tested and is considered stable for production environments.

Please see the [SvelteKit 1.0 release notes](https://svelte.dev/blog/announcing-sveltekit-1.0) for detailed information on what this upgrade offers.

## Types of changes

This code introduces the following type of change: 

- [ ] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [x] Bug Fix (non-breaking change which fixes an issue).
- [x] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -2,5 +2,8 @@
 	"useTabs": true,
 	"singleQuote": true,
 	"trailingComma": "none",
-	"printWidth": 100
+	"printWidth": 100,
+	"plugins": ["prettier-plugin-svelte"],
+	"pluginSearchDirs": ["."],
```
Comment (flicksolutions) [inline .prettierrc:7]: Ich schlage vor, prettier auf die nächste Major-version upzudaten.
Ab 3.0 wurde pluginSearchDirs dann entfernt: https://prettier.io/blog/2023/07/05/3.0.0.html#plugin-search-feature-has-been-removed-14759httpsgithubcomprettierprettierpull14759-by-fiskerhttpsgithubcomfisker

File: package.json [hunk 2]
```diff
@@ -4,35 +4,34 @@
...
-		"mdsvex": "^0.9.8",
-		"prettier": "~2.2.1",
-		"prettier-plugin-svelte": "^2.6.0",
-		"svelte": "^3.46.4",
-		"svelte-check": "^2.4.6",
-		"svelte-preprocess": "^4.10.4",
-		"tslib": "^2.3.1",
-		"typescript": "^4.6.2"
+		"eslint-plugin-svelte": "^2.30.0",
+		"prettier": "^2.8.0",
```
Comment (flicksolutions) [inline package.json:27]: I propose ^3.0.1
following prettier-plugin-svelte and eslint-config-prettier have to be updated aswell.

File: playwright.config.js
Comment (flicksolutions) [inline playwright.config.js:1]: Can't say much about playwright... I've never used it.
I just assume this is right.

File: src/routes/onboarding/+page.svelte [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <script>
 	import { assets } from '$app/paths';
-	import About from '../lib/components/About.svelte';
-	import Videos from '../lib/components/Videos.svelte';
+	import About from '../../lib/components/About.svelte';
```
Comment (flicksolutions) [inline src/routes/onboarding/+page.svelte:3]:
```suggestion
	import About from '$lib/components/About.svelte';
```

File: src/routes/onboarding/+page.svelte [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+	import Videos from '../../lib/components/Videos.svelte';
```
Comment (flicksolutions) [inline src/routes/onboarding/+page.svelte:4]: dito

File: src/routes/onboarding/+page.svelte
Comment (flicksolutions) [inline src/routes/onboarding/+page.svelte:1]: `/videos/speakers.png` is linked on line 50. this file doesn't exist. I propose to rename `/videos/speaker.png` to `/videos/speakers.png`

**Review by flicksolutions — CHANGES_REQUESTED** [review]
Everything looks really fine! The things I found are minor.

File: mdsvex.config.js [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
 	smartypants: {
 		dashes: 'oldschool'
 	},
+
 	layout: '/src/lib/components/MdLayout.svelte',
+	remarkPlugins: [],
+	rehypePlugins: [],
+	extensions: ['.svelte.md', '.md', '.svx'],
+
+	smartypants: {
```
[the remark this answers is not in the document]
Comment (maehr) [reply mdsvex.config.js:14]: Duplication removed

File: svelte.config.js [hunk 6]
```diff
@@ -1,24 +1,19 @@
 import { mdsvex } from 'mdsvex';
 import mdsvexConfig from './mdsvex.config.js';
-import preprocess from 'svelte-preprocess';
-import adapter from '@sveltejs/adapter-cloudflare';
+import adapter from '@sveltejs/adapter-auto';
```
[the remark this answers is not in the document]
Comment (maehr) [reply svelte.config.js:3]: Auto also works for cloudflare (if I am not mistaken)

File: mdsvex.config.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (maehr) [reply mdsvex.config.js]: removed

**Review by flicksolutions — APPROVED** [review]
approved! Well done!

**Pull request merged**